common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -40,7 +36,30 @@
 # Never produce more than this many different seeds, no matter what.
 MAX_RANDOMIZATION_BINS = 1000
 
-FEATURES = getattr(settings, 'FEATURES', {})
+
+class SHOWANSWER(object):
```

Comment:
Moved over from `common/lib/xmodule/xmodule/capa_base_constants.py` to reduce file clutter.